packages/react-query/src/scripts/utils/table.ts

```diff
@@ -1,15 +1,15 @@
 import Table from 'cli-table3'
 import { getPackageJson, getSuspensiveReactQueryPackageJson, getTanStackReactQueryPackageJson } from './package'
 
-export const VERSION5_APIS = [
+const VERSION5_APIS = [
   '<SuspenseQuery/>',
   '<SuspenseQueries/>',
   '<SuspenseInfiniteQuery/>',
   '<Mutation/>',
   '<QueryErrorBoundary/>',
 ]
 
-export const VERSION4_APIS = [
+const VERSION4_APIS = [
```

Comment:
@gwansikk Check this out please. knip detect unused variables👍 implemented by @SEOKKAMONI